### ISSUE TITLE
Define more than one mail recepient with MailJet and SendGrid

### ISSUE THREAD

**Issue by stoyanovgeorge** [orangecoding/fredy#33]
I would like to add more than one mail recipient and tried the following separators between them:

- space
- comma
- semicolon 

None of them worked and if you have created one notification adapter of type A, Fredy doesn't allow you to create another one of the same type.

**Comment by orangecoding** [comment]
Interesting. At around 5k downloads, I never had such a request, even tho it makes sense. Will take a Look, thanks

**Comment by orangecoding** [comment]
@stoyanovgeorge I've created a pr for this. Can you please give it a review and a thumbs up if cool with you?

**Comment by orangecoding** [comment]
By the way, I accept pr's.. next time you can add this yourself. just saying ;)

**Comment by stoyanovgeorge** [comment]
I will test it tomorrow and update you here if there is a problem. Thanks a lot for your quick reaction. 
Cheers!

**Comment by orangecoding** [comment]
@stoyanovgeorge did you have a chance to test this?

**Comment by stoyanovgeorge** [comment]
I tried to install it yesterday on my RPI4B and create a docker container and run it from there, but the problem was that in the process I have somehow managed to crash my board so hard, that it didn't respond after the restart and I was not able to connect to it over SSH, so I have stopped. I will try to recover the board today and see if it would work. My problem was that in the past, the fredy stopped working after a while and I decided to give Docker containers a try.
